official/docs/java/addresses/CreateAddress.java

```diff
@@ -0,0 +1,22 @@
+import com.easypost.EasyPost;
+
+public class CreateAddress {
+    public static void main(String[] args) {
+        EasyPost.apiKey = "<EASYPOST_API_KEY>";
```

Comment:
We should be using environment variables on all of these please.

```suggestion
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
```